db/queries.py

```diff
@@ -1,23 +1,38 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+import datetime
+
+from django.db.models import QuerySet, Q, F
+from db.models import Customer, LoyaltyProgram, LoyaltyProgramParticipant
 
 
 def all_loyalty_program_names() -> QuerySet:
-    pass
+    return LoyaltyProgram.objects.all().values_list("name", "bonus_percentage")
 
 
 def not_active_customers() -> QuerySet:
-    pass
+    query = LoyaltyProgramParticipant.objects.all()
+    dt1 = datetime.date(2021, 1, 1)
+    dt2 = datetime.date(2022, 1, 1)
+    name_list = query.filter(last_activity__gt=dt1, last_activity__lt=dt2)
+    return name_list.values("customer__first_name")
 
 
 def most_active_customers() -> QuerySet:
-    pass
+    active_cust = LoyaltyProgramParticipant.objects.all()
+    ordered = active_cust.order_by("-sum_of_spent_money")[:5]
```

Comment:
No need to write objects.all() here, u can make LoyaltyProgramParticiapant.objects.order_by(...)